install_config/install/deploy_router.adoc

```diff
@@ -265,6 +265,193 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
+
+|`a`
+|`high`
+|`east`
+|`modest`
+|`finance`
+|`type=static`
+
+|`b`
+|
+|`west`
+|`strong`
+|
+|`type=dynamic`
+
+|`c`, `d`, `e`
+|`low`
+|
+|`modest`
+|
+|`type=static`
+
+|`g` -- `k`
+|`medium`
+|
+|`strong`
+|`dev`
+|
+
+|`l` -- `s`
+|`high`
+|
+|`modest`
+|`ops`
+|
+
+|`t` -- `z`
+|
+|`west`
+|
+|
+|`type=dynamic`
+
+|===
+
+Here is a convenience script *_mkshard_*  that
+centralizes the common arguments of the `oadm router` command:
+
+====
+[source,bash]
+----
+#!/bin/bash
+# Usage: mkshard ID SELECTION-EXPRESSION
+id=$1
+sel="$2"
+oadm router router-shard-$id \//<1>
+    --selector="$sel" \//<2>
+    --replicas=3 \
+    --credentials=/etc/origin/master/openshift-router.kubeconfig \
+    --service-account=ipfailover
+exit $?
+----
+<1> The created router has name `router-shard-<id>`.
+<2> Specify the selection expression with `--selector`.
+====
+
+Running *_mkshard_* several times creates several routers:
+
+[options="header",cols="2,3,2"]
+|===
+|Router |Selection Expression |Routes
+
+|`router-shard-1`
+|`sla=high`
+|`a`, `l` -- `s`
+
+|`router-shard-2`
+|`geo=west`
+|`b`, `t` -- `z`
+
+|`router-shard-3`
+|`dept=dev`
+|`g` -- `k`
+
+|===
+
+
+[[modifying-router-shards]]
+=== Modifying Router Shards
+
+Because a router shard is a construct
+link:../../architecture/core_concepts/routes.html#router-sharding[based on labels],
+you can modify either the labels or the selection expression.
+
+This section extends the example started in the
+link:#creating-router-shards[Creating Router Shards] section,
+demonstrating how to change the selection expression.
+// TODO:XREF modifying labels
+
+Here is a convenience script *_modshard_* that modifies
+an existing router to use a new selection expression:
+
+====
+[source,bash]
+----
+#!/bin/bash
+# Usage: modshard ID SELECTION-EXPRESSION
+id=$1
+sel="$2"
+router=router-shard-$id       //<1>
+dc=dc/$router                 //<2>
+oc scale $dc --replicas=0     //<3>
+oc edit  $dc "$sel"           //<4>
+oc scale $dc --replicas=3     //<5>
+exit $?
+----
+<1> The modified router has name `router-shard-<id>`.
+<2> The deployment configuration where the modifications occur.
+<3> Scale it down.
+<4> Set the new selection expression.
+<5> Scale it back up.
+====
+
+For example, to expand the department for `router-shard-3`
+to include `ops` as well as `dev`:
+
+----
+$ modshard 3 ROUTE_LABELS='dept in (dev, ops)'
+----
+
+The result is that `router-shard-3` now selects routes `g` -- `s`
+(the combined sets of `g` -- `k` and `l` -- `s`).
+
+This example takes into account that
+there are only three departments in this example scenario,
+and specifies a department to leave out of the shard,
+thus achieving the same result as the preceding example:
+
+----
+$ modshard 3 ROUTE_LABELS='dept != finanace'
+----
+
+This example specifies shows three comma-separated qualities,
+and results in only route `b` being selected:
+
+----
+$ modshard 3 hw=strong,type=dynamic,geo=west
```

Comment:
Typo missing environment variable name for `hw=strong,type=dynamic,geo=west`.  
Ala: `ROUTE_LABELS="hw=strong,type=dynamic,geo=west"`